Trappist/src/Promact.Trappist.Test/Tests/TestsRepositoryTest.cs

```diff
@@ -106,11 +106,36 @@ public TestsRepositoryTest(Bootstrap bootstrap) : base(bootstrap)
             await _testRepository.CreateTestAsync(test);
             var testNameToUpdate = await _testRepository.GetTestSettingsAsync(test.Id);
             testNameToUpdate.TestName = "MCKV";
-            await _testRepository.UpdateTestNameAsync(test.Id,testNameToUpdate);
+            await _testRepository.UpdateTestNameAsync(test.Id, testNameToUpdate);
             var TestName = "MCKV";
             Assert.True(_trappistDbContext.Test.Count(x => x.TestName == TestName) == 1);
         }
 
+        /// <summary>
+        /// Check if any test attendee exist for a particular test
+        /// </summary>
+        [Fact]
+        public async Task IsAttendeeExistsAsync()
```

Comment:
Missing scenarios of delete test cases. identify and write them. All the conditions you have checked should be covered.